packages/components/src/tools-panel/stories/index.js

```diff
@@ -76,7 +77,103 @@ export const _default = () => {
 	);
 };
 
+const { Fill, Slot } = createSlotFill( 'ToolsPanelSlot' );
+
+const ToolsPanelItems = ( { children } ) => {
+	return <Fill>{ children }</Fill>;
+};
```

Comment:
Would it work as follows?
```suggestion
const { Fill: ToolsPanelItems, Slot } = createSlotFill( 'ToolsPanelSlot' );
```

Reply:
It does work although I get a warning about the `Slot` property not existing on type. Which can be avoided by simply using the `Slot` direct from `createSlotFill`. I was trying to stay consistent with how most other SlotFills are handled in other packages.

I'll update this so the code is as simple as possible.

Reply:
✅ Done. Thanks for the suggestion.